separate the two Zendesk pipelines sharing this repository.

szd55gds opened the pull request:
Currently a merge to this repository will trigger both pipelines. That is undesirable for the deduplicator because there are manual followup activities carried out by the support team after it runs. These changes will separate the 2 tasks such that only a merge to the respective code will trigger the pipeline.

Updated README with explanation.

blairboy362 commented on ci/pipeline.yml:
```diff
@@ -5,6 +5,9 @@ resources:
     branch: master
     uri: "https://github.com/alphagov/zendesk-scripts.git"
+    ignore_paths:
+    - /usr/src/app/lib/zendesk-ticket-deduplicator.rb
```
I think these should be relative paths in the repository tree.

blairboy362 replied:
```suggestion
    - lib/zendesk-ticket-deduplicator.rb
```

blairboy362 commented on ci/zendesk_de-duplicator.yml:
```diff
@@ -5,6 +5,9 @@ resources:
     branch: master
     uri: "https://github.com/alphagov/zendesk-scripts.git"
+    paths:
+    - /usr/src/app/lib/zendesk-ticket-deduplicator.rb
```
```suggestion
    - lib/zendesk-ticket-deduplicator.rb
```

chrisfarms commented on README.md:
```diff
@@ -253,6 +253,15 @@ window_start_time = Time.now - 24 * 3600
+
+##### Important Note
+
+This pipeline should only run 6 hourly in order to mesh with the manual processes carried out by the support team. If updating the code, please pause the pipeline to ensure it does not run between the 6 hour windows.
```
Can you add the "why" here ... ie "running outside of the window may cause unexpecting moving of tickets around during the hours people are working on them" .... so that future people who _need_ to make a change know what the impact will be.

szd55gds replied:
Thanks @chrisfarms embodied your suggestion.

szd55gds commented:
@blairboy362 Hi Daniel, I've added changes to move all the paths to relative as you kindly recommended. I may have muddled up the github comments but hope I've addressed all your points.
